tests/testApproximate.py

```diff
@@ -53,7 +53,8 @@ def makeRamp(self, binsize=1):
         rampCoeffs = (1000, 1, 1)
         for i in range(ramp.getHeight()):
             for j in range(ramp.getWidth()):
-                ramp.set(j, i, (rampCoeffs[0] + rampCoeffs[1]*x[j] + rampCoeffs[2]*y[i], 0x0, var))
+                ramp.set(
+                    j, i, (rampCoeffs[0] + rampCoeffs[1]*x[j] + rampCoeffs[2]*y[i], 0x0, var))
```

Comment:
I think the code is readable enough. Surely we can live with autopep8's choices even if they aren't optimal? Unless the results are truly egregious, hand-formatting code is a waste of developer time. Furthermore autopep8 could be run again at any time, so any hand tuning is likely to be undone at a later point.